kafka/4.6/modules/ROOT/pages/index.adoc

```diff
@@ -28,6 +28,60 @@ An example is a newsroom that uses the Apache Kafka system to deliver the latest
+** To optimize for performance, companies can batch messages to send. Kafka stores the message until the batch size is reached and completes the send at that time.
+* Configuring the message size to send to focus on optimal performance.
+** Kafka is not intended to process and handle large messages. Messages are typically in the 1 MB range and users set the size to avoid performance degradation. In the case of an error, the message is rejected.
+* Blocking further commits on a configurable level of acknowledgment for the messages published to the Kafka server to ensure successful data transmission.
+** Acknowledgments enable users to ensure that the message is sent prior to publishing more messages. This is specifically useful in time-ordered or event-ordered messages, such as logs or patient drug administration orders. Ensuring that messages are sent in order helps the downstream application process the messages in the right order.
+* Configuring how data is serialized during a publish.
+* Securing messages sent during Mutual TLS.
+
+The following lists the acceptance criteria for publishing:
+
+* Publish a message to a Kafka topic.
+* Configure the batch size and size of the message.
+* Obtain an acknowledgment once publishing is complete at the configured level.
+* Send the message securely via Mutual TLS.
+* Configure the serialization type.
+
+=== Subscribe-from Use Cases
+
+With Kafka Connector, applications can obtain messages from Kafka for further consumption downstream. Some examples of use cases include:
+
+Reading from Kafka:
+
+* Subscribing to the Kafka server and the underlying group to read and consume messages for downstream processing.
+* Subscribing to a specific Kafka partition to process messages.
+** Users with a high availability microservice don't want to rely on Kafka's rebalancing in the event of a failure.
+** If users have a local state associated with their partition, they want to process only that partition.
+* Configuring the offset value to a set value or the beginning or the end of the topic.
+** Users have a local store and want to use that information to start processing their records.
+** Users have time-sensitive records and want to skip to the end if the application must be up-to-date or skip to the beginning if it is a new application processing the records.
+* Reading the offset value from an alternate storage system on restart.
+** Users have a local store with the offset information and want to read the value from their local storage.
+* Setting the interval for which to block the receipt of messages for varying downstream processing intervals.
+
+Processing the messages:
+
+* Ensuring there is no data loss when the downstream application crashes and manually configuring commits to have more control.
+* Storing a commit value in a local store or in the Kafka storage and blocking until the store is successful.
+* Storing an offset in a local store or in the Kafka storage asynchronously.
```

Comment:
I think this should be:
Storing an offset asynchronously in a local store or in the Kafka storage